helm-chart/rode/templates/ingress.yaml

```diff
@@ -5,10 +5,15 @@ kind: Ingress
 metadata:
   name: {{ include "rode.fullname" . }}
   labels:
-    app: {{ template "rode.name" . }}                                                                                                                         
+    app: {{ template "rode.name" . }}
     helm.sh/chart: {{ .Chart.Name }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
+  annotations:
+    description: Ingress for Rode
```

Comment:
is there a reason you're adding a `description` annotation by default?  I don't think I've seen other helm charts doing this

Reply:
It was a pattern I saw here in the other `lead-sdm-operators`: https://github.com/liatrio/lead-sdm-operators/blob/master/charts/operator-toolchain/templates/ingress.yaml#L18

Reply:
gotcha.  since this doesn't seem to be something any other public chart is doing, let's just remove this.